client/src/main/java/org/apache/celeborn/client/ReviveManager.java

```diff
@@ -124,4 +123,8 @@ public void addRequest(ReviveRequest request) {
       logger.error("Exception when put into requests!", e);
     }
   }
+
+  public void close() {
+    batchReviveRequestScheduler.shutdown();
```

Comment:
should we shutdown immediately or give it a meaningful timeout, otherwise it may block the shutdown procedure.

Hadoop MRG has default 30s, all shutdown hooks share that threshold, one hook blocking will cause the subsequent hooks skip to execute